server/sonar-web/src/main/js/apps/projects/filters/LanguagesFilter.js

```diff
@@ -54,13 +59,13 @@ export default class LanguagesFilter extends React.PureComponent {
       .map(key => ({ label: languages[key].name, value: key }));
   }
 
-  getSortedOptions(facet: {} = {}) {
+  getSortedOptions(facet /*: {} */ = {}) {
     return sortBy(Object.keys(facet), [option => -facet[option], option => option]);
   }
 
-  getFacetValueForOption = (facet: {} = {}, option: string) => facet[option];
+  getFacetValueForOption = (facet /*: {} */ = {}, option /*: string */) => facet[option];
 
-  renderOption = (option: string) =>
+  renderOption = (option /*: string */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)